lib/lightning_web/live/project_live/settings.ex

```diff
@@ -579,6 +589,16 @@ defmodule LightningWeb.ProjectLive.Settings do
     """
   end
 
+  def checked?(changeset, id) do
+    case Ecto.Changeset.fetch_field(changeset, :retention_policy) do
+      {_changes_data, ^id} ->
```

Comment:
I don't understand the use of the term `id` here, is it not just `value`?

Also I think we can safely use `fetch_field!/2` here, that would also tighten this up to: 

```suggestion
Ecto.Changeset.fetch_field!(changeset, :retention_policy) == value
```

Reply:
From the UI side, `id` here would mean the `input_id` or input field while the value comes from the changeset. But you're right about using `fetch_field!`, we don't need to know anymore if the value comes from changes or data.